easybuild/easyconfigs/v/Valgrind/Valgrind-3.15.0-intel-2020a.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Valgrind'
+version = '3.15.0'
+
+homepage = 'https://valgrind.org'
+description = "Valgrind: Debugging and profiling tools"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'optarch': False}
+
+source_urls = [
+    'ftp://sourceware.org/pub/valgrind/',
+    'https://www.mirrorservice.org/sites/sourceware.org/pub/valgrind/',
+]
+sources = [SOURCELOWER_TAR_BZ2]
+patches = ['Valgrind-3.15.0-no-MPI-1.patch']
```

Comment:
This patch is technically speaking only required in the foss-2020a build., please remove it from the intel build.

Reply:
done